package.json

```diff
@@ -43,7 +43,8 @@
 	],
 	"main": "./out/extension.js",
 	"activationEvents": [
-		"onNotebook:*"
+        "onNotebook:jupyter-notebook",
+        "onNotebook:interactive"
```

Comment:
I don't think we should activate for all notebooks e.g. github issues notebooks, kusto, etc

Reply:
I'm concerned that we'd end up activating the Jupyter extension unnecessary as well.
E.g. open a github issues notebook and now we'll pull in Jupyter extension as a result of this, hence jupyter too gets activated.

Reply:
I guess we can worry about that later, i know some will not like that.

Reply:
That's true, I wasn't thinking about the dependency. We should probably restrict this. I'll do that.